### PR TITLE
Added handling custom metrics to resource consumer.

### PR DESCRIPTION
**Pull request by jszczepkowski**
Added handling custom metrics in Prometheus format to resource consumer.

### CODE REVIEW

File: test/images/resource-consumer/resource_consumer_handler.go [hunk 1]
```diff
@@ -116,3 +140,54 @@ func (handler ResourceConsumerHandler) handleGetCurrentStatus(w http.ResponseWri
...
 }
+
+func (handler ResourceConsumerHandler) handleMetrics(w http.ResponseWriter) {
+	handler.metricsLock.Lock()
+	for k, v := range handler.metrics {
+		fmt.Fprintf(w, "# HELP %s info message.\n", k)
+		fmt.Fprintf(w, "# TYPE %s gauge\n", k)
+		fmt.Fprintf(w, "%s %f\n", k, v)
+	}
+	handler.metricsLock.Unlock()
```
Comment (piosz) [inline test/images/resource-consumer/resource_consumer_handler.go]: Move this to defer right after the line where the lock is acquired .

**Comment by piosz** [comment]
LGTM, just a small nit

File: test/images/resource-consumer/resource_consumer_handler.go (hunk 1 above)
Comment (jszczepkowski) [inline test/images/resource-consumer/resource_consumer_handler.go]: Done.

**Comment by jszczepkowski** [comment]
PTAL

**Pull request merged**